crates/sui-json-rpc/src/unit_tests/rpc_server_tests.rs

```diff
@@ -44,7 +44,14 @@ async fn test_get_objects() -> Result<(), anyhow::Error> {
     let http_client = cluster.rpc_client();
     let address = cluster.accounts.first().unwrap();
 
-    let objects = http_client.get_objects_owned_by_address(*address).await?;
+    let objects = http_client
+        .get_objects_owned_by_address(
+            *address,
+            Some(SuiObjectDataOptions::full_content()),
```

Comment:
nit: I know this is a test, but it would be good to use `SuiObjectDataOptions::new()` here since it does not need full content at all